pkgs/top-level/python-packages.nix

```diff
@@ -17329,6 +17353,29 @@ in {
     };
   };
 
+  unittest-xml-reporting = buildPythonPackage rec {
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.